week1_HW/todo-app/src/components/task-list.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import Tasks from "./task"
```

Comment:
By convention, all React component files should be named exactly the same as the components themselves. So you need to rename these files as `TaskList.js`, `Task.js`, etc.

Also, you're importing the `Task` component as `Tasks` which is not accurate - it only displays a single task.